Update gRPC security bbes (#3812)
ballerina-platform/ballerina-distribution: examples/grpc-client-bearer-token-auth/grpc_client_bearer_token_auth.md

# gRPC client - Bearer token authentication

The `grpc:Client` can connect to a service that is secured with bearer token authentication by enriching the client metadata with the `Authorization: Bearer <token>` header. The bearer token can be specified in the `auth` field of the client configuration. Use this to communicate with the service, which is secured with bearer token authentication.

   ::: code grpc_client_bearer_token_auth.bal :::

> You can refer to the [gRPC client - Unary RPC](/learn/by-example/grpc-client-unary/) to implement the client used here.

Reviewer: ```suggestion
For information on implementing the client, see [gRPC client - Unary RPC](/learn/by-example/grpc-client-unary/).
```
This is the format in which hyperlinks should be included. Please check and fix all occurrences.
